packages/@react-stately/color/src/useColorWheelState.ts

```diff
@@ -32,9 +32,9 @@ export interface ColorWheelState {
   getThumbPosition(radius: number): {x: number, y: number},
 
   /** Increments the hue by the given amount (defaults to 1). */
-  increment(minStepSize?: number): void,
+  increment(isPage?: boolean): void,
   /** Decrements the hue by the given amount (defaults to 1). */
-  decrement(minStepSize?: number): void,
+  decrement(isPage?: boolean): void,
```

Comment:
Isn't this a breaking change?

Reply:
yeah, though it's beta, so if we wanted to do this, now would be the time
I think this makes more sense, aria hooks don't care what the step sizes are, but the state very much cares

Reply:
hmm, I don't particularly like passing booleans as parameters though. it's not very extensible for future use cases, and doesn't allow users of stately to override the page size. If we want to keep it in stately, I would make a separate method (e.g. `incrementStep` - we have `incrementPage` in datepicker, but in this case they aren't really "pages"). But I also think it would be fine and more flexible to just pass the step size in.

Reply:
good point, i'll see what we can do about it